lib/components/src/treeview/treeview.js

```diff
@@ -0,0 +1,301 @@
+import React, { PureComponent } from 'react';
```

Comment:
I'm hoping to extract this component into it's own package and publish as `react-tree`